### PR TITLE
Fixed merge conflicts and added date basket

### PR DESCRIPTION
**Pull request by AndrewBrooking**

### CODE REVIEW

File: assets/scripts/queries.js [hunk 1]
```diff
@@ -1,85 +1,85 @@
...
-
-var searchArea = "29.7604,-95.3698"; //needs to be a latlng generated from user inputted zipcode
-date = "2019-08-09"; //date in form YYYY-MM-DD
-var songkickResultsArr = [];
-let locSearchURL = "https://api.songkick.com/api/3.0/search/locations.json?" + songkickApiKey + "&location=geo:" + searchArea;
-musicSearch(locSearchURL);
 
+// ############################### Google Places API Functions ###############################
 
-//function does an api call to get songkick metro id from latlng. then calls function to do upcoming events search
```
Comment (meisly) [inline assets/scripts/queries.js]: This is necessary

**Review by meisly — CHANGES_REQUESTED** [review]
queries is a cluster

File: assets/scripts/queries.js (hunk 1 above)
Comment (AndrewBrooking) [reply assets/scripts/queries.js]: Fixed

**Review by meisly — CHANGES_REQUESTED** [review]
Add Bladimirs changes from master

File: assets/scripts/queries.js [hunk 2]
```diff
@@ -1,10 +1,80 @@
...
+                name: name,
+                location: location,
+                type: type,
+                icon: icon,
+            }
+
+            // 
+            gSearchResultsARR.push(gSearchResultOBJ);
+        }
+
```
Comment (meisly) [inline assets/scripts/queries.js:86]: Bladimir's code should show up here but it isn't

**Review by meisly — APPROVED** [review]

**Pull request merged**